### PR TITLE
[NWPS-1411-1424-1425-1427] Adding compounds to Content Blocks - News & Blogs

### PR DESCRIPTION
**Pull request by sankar-shunmuga**
Please review this ticket

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/namespaces/hee/blogPage.yaml [hunk 1]
```diff
@@ -299,7 +299,7 @@ definitions:
           /rightHandBlocks:
             jcr:primaryType: frontend:plugin
             caption: Right hand objects
-            compoundList: hee:contactCardWithDescriptionReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:internalLinksCardReference,hee:QuickLinks
+            compoundList: hee:contactCardWithDescriptionReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:internalLinksCardReference,hee:QuickLinks
```
Comment (smanivasag) [inline repository-data/application/src/main/resources/hcm-config/namespaces/hee/blogPage.yaml:302]: Could you add the rendering support for `hee:ctaCardReference` on `repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/blogpost-main.ftl`:
```
<#case "uk.nhs.hee.web.beans.CtaCardReference">
    <@hee.ctaCard ctaCard=block/>
    <#break>
```

File: repository-data/application/src/main/resources/hcm-config/namespaces/hee/news.yaml [hunk 2]
```diff
@@ -266,7 +266,7 @@ definitions:
           /rightHandBlocks:
             jcr:primaryType: frontend:plugin
             caption: Right hand objects
-            compoundList: hee:contactCardWithDescriptionReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:QuickLinks
+            compoundList: hee:contactCardWithDescriptionReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:internalLinksCardReference,hee:QuickLinks
```
Comment (smanivasag) [inline repository-data/application/src/main/resources/hcm-config/namespaces/hee/news.yaml:269]: Could you add the rendering support for `hee:internalLinksCardReference` on `repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/news-main.ftl`:
```
<#case "uk.nhs.hee.web.beans.InternalLinksCardReference">
    <@hee.internalLinksCard card=block.internalLinksCard/>
    <#break>
```

File: repository-data/application/src/main/resources/hcm-config/namespaces/hee/blogPage.yaml (hunk 1 above)
Comment (sankar-shunmuga) [reply repository-data/application/src/main/resources/hcm-config/namespaces/hee/blogPage.yaml:302]: Done

**Review by smanivasag — APPROVED** [review]
@sankar-shunmuga - Thanks for the PR and is now approved 👍

**Pull request merged**